src/app/components/AccountInfo/DepositModal.tsx

```diff
@@ -41,9 +42,15 @@ class DepositModal extends React.Component<Props> {
       isOpen,
       onClose,
       hasPassword,
+      nodeInfo,
     } = this.props;
     const isVisible = !!isOpen && !!(hasPassword || fetchDepositAddressError);
 
+    let blockchain = "Bitcoin";
+    if (nodeInfo && nodeInfo.chains[0] === 'litecoin') {
```

Comment:
will nodeInfo ever not exist?

Reply:
if it's always there, then just set the name to nodeInfo.chains[0] capitalized.

Reply:
I thought about doing this, but IntelliJ warns that nodeInfo could possible be null.